R/utils.R

```diff
@@ -331,7 +331,7 @@ header <- function(label,
                    suffix = "-",
                    n = min(getOption("width"), 78))
 {
-  n <- n - nchar(label) - nchar(prefix) - 2L
+  n <- max(n - nchar(label) - nchar(prefix) - 2L, 3)
```

Comment:
This ensures there will always be at least three `---` following the header text?

Reply:
Yeah, I don't think it makes much difference for real life, but guarantees that there's something unique for the transformer to latch on to.